apps/addons/decorators.py

```diff
@@ -10,6 +10,12 @@
 log = commonware.log.getLogger('mkt.purchase')
 
 
+def owner_or_unlisted_reviewer(request, addon):
+    return (acl.check_unlisted_addons_reviewer(request) or
+            acl.check_addon_ownership(request, addon, admin=False, dev=True,
```

Comment:
Admins, in this context, is simply anyone which has "Addons:Edit" acces (which is also the case for simple reviewers, I believe).

We only want the group that specifically has access (is in the "unlisted addons reviewers" group), or is a superuser, to be able to see those.